lib-utilities/config/constants.go

```diff
@@ -90,6 +93,12 @@ const (
 	DefaultTLSMaxVersion = tls.VersionTLS12
 	// DefaultTLSServerVerify - indicator for performing server validation
 	DefaultTLSServerVerify = true
+	// DefaultDeliveryRetryAttempts - default DeliveryRetryAttempts value
+	DefaultDeliveryRetryAttempts = 3
+	// DefaultDeliveryRetryIntervalSeconds - default DeliveryRetryIntervalSeconds value
+	DefaultDeliveryRetryIntervalSeconds = 60
+	// DefaultRetentionEventsInMinutes - default RetentionEventsInMinutes value
+	DefaultRetentionEventsInMinutes = 10
```

Comment:
Rename it to `DefaultRetentionOfUndeliveredEventsInMinutes`

Reply:
changed